### PR TITLE
Feat: Slogan section v2

### PR DESCRIPTION
**Pull request by Penggeor**

### CODE REVIEW

**Comment by Penggeor** [comment]
#20

File: src/css/custom.css [hunk 1]
```diff
@@ -6,6 +6,26 @@
...
 
+@font-face {
+  font-family: 'Jost';
+  src: url("/static/fonts/Jost-Light.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: face;
```
Comment (92hackers) [inline src/css/custom.css:17]: `font-family: face;`, what's this ?

**Review by 92hackers — CHANGES_REQUESTED** [review]

File: src/css/custom.css (hunk 1 above)
Comment (Penggeor) [reply src/css/custom.css:17]: This is mistake.

File: src/utils/MathPlus.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+function limitInRange(min: number, max: number, value: number): number {
+  switch (true) {
```
Comment (92hackers) [inline src/utils/MathPlus.ts:2]:
```javascript
if (value < min) return min
if (value > max) return max
return value
```

**Pull request merged**